packages/server/__snapshots__/8_read_only_fs.ts.js

```diff
@@ -0,0 +1,75 @@
+Folder /tmp/8_read_only_fs is not writable.
+
+Writing to this directory is required by Cypress in order to store screenshots and videos.
+
+Enable write permissions to this directory to ensure screenshots and videos are stored. 
+
+If you don't require screenshots or videos to be stored you can safely ignore this warning.
+
+====================================================================================================
+
+  (Run Starting)
+
+  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ Cypress:    1.2.3                                                                              │
+  │ Browser:    FooBrowser 88                                                                      │
+  │ Specs:      1 found (spec.js)                                                                  │
+  │ Searched:   cypress/integration/spec.js                                                        │
+  └────────────────────────────────────────────────────────────────────────────────────────────────┘
+
+
+────────────────────────────────────────────────────────────────────────────────────────────────────
+                                                                                                    
+  Running:  spec.js                                                                         (1 of 1)
+
+
+  ✓ test
+
+  1 passing
+
+
+  (Results)
+
+  ┌────────────────────────────────────────────────────────────────────────────────────────────────┐
+  │ Tests:        1                                                                                │
+  │ Passing:      1                                                                                │
+  │ Failing:      0                                                                                │
+  │ Pending:      0                                                                                │
+  │ Skipped:      0                                                                                │
+  │ Screenshots:  0                                                                                │
+  │ Video:        true                                                                             │
+  │ Duration:     X seconds                                                                        │
+  │ Spec Ran:     spec.js                                                                          │
+  └────────────────────────────────────────────────────────────────────────────────────────────────┘
+
+Warning: We failed to record the video.
+
+This error will not alter the exit code.
+
+Error: ffmpeg exited with code 1: /tmp/8_read_only_fs/cypress/videos/spec.js.mp4: Permission denied
+
+      [stack trace lines]
+
+Warning: We failed processing this video.
+
+This error will not alter the exit code.
+
+Error: ffmpeg exited with code 1: /tmp/8_read_only_fs/cypress/videos/spec.js.mp4: Permission denied
+
+      [stack trace lines]
```

Comment:
would also like to see a screenshot failure here... using a spec that fails should generate a screenshot